### PR TITLE
Part16 all tasks done

### PR DESCRIPTION
**Pull request by alexkulesh**

### CODE REVIEW

File: FirstSolution/Part16Task1/ISwimable.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Part16Task1
+{
+    public interface ISwimable
+    {
+        void Swim()
+        {
+        }
```
Comment (Alex19777) [inline FirstSolution/Part16Task1/ISwimable.cs]: Remove brackets, they need to specify default realization

File: FirstSolution/Part16Task1/Makrel.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Part16Task1
+{
+    public class Makrel : Fish
+    {
```
Comment (Alex19777) [inline FirstSolution/Part16Task1/Makrel.cs:6]: Add some members

File: FirstSolution/Part16Task2/Ball.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Part16Task2
+{
+    public class Ball
+    {
+        public void Move(string position)
+        {
+            Console.WriteLine($"мяч в {position}");
```
Comment (Alex19777) [inline FirstSolution/Part16Task2/Ball.cs]: Translate to English please

File: FirstSolution/Part16Task2/FieldPlayer.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+namespace Part16Task2
+{
+    public class FieldPlayer : Person, IPlayer
+    {
+        private int counter;
+        public FieldPlayer(string name) : base(name)
+        {
+            Name = name;
+            Console.WriteLine($"Сегодня играет {Name}");
```
Comment (Alex19777) [inline FirstSolution/Part16Task2/FieldPlayer.cs]: same

File: FirstSolution/Part16Task1/ISwimable.cs (hunk 1 above)
Comment (alexkulesh) [reply FirstSolution/Part16Task1/ISwimable.cs]: Done

File: FirstSolution/Part16Task1/Makrel.cs (hunk 2 above)
Comment (alexkulesh) [reply FirstSolution/Part16Task1/Makrel.cs:6]: Done

File: FirstSolution/Part16Task2/Ball.cs (hunk 3 above)
Comment (alexkulesh) [reply FirstSolution/Part16Task2/Ball.cs]: Done

File: FirstSolution/Part16Task2/FieldPlayer.cs (hunk 4 above)
Comment (alexkulesh) [reply FirstSolution/Part16Task2/FieldPlayer.cs]: Done

File: FirstSolution/Part16Task2/IPlayer.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+        public void Pass(FieldPlayer player)
+        {
+
+        }
+
+        public void PickBooking()
+        {
+            
+        }
```
Comment (Alex19777) [inline FirstSolution/Part16Task2/IPlayer.cs:18]: No need to put method body here

**Review by Alex19777 — APPROVED** [review]

File: FirstSolution/Part16Task2/IPlayer.cs (hunk 5 above)
Comment (alexkulesh) [reply FirstSolution/Part16Task2/IPlayer.cs:18]: Done

**Pull request merged**